SystemInfo/Value/ComposerPackage.php

```diff
@@ -34,6 +34,16 @@ class ComposerPackage extends ValueObject implements SystemInfo
     public $version;
 
     /**
+     * Stability.
+     *
+     * String with integer, as set in composer.lock "stability-flags".
```

Comment:
Well, the tests are not complicated by this at the moment, though I am sure we can find ways to make them so :)
But first let's see if the stability feature/hack is something you guys want, given the hardcoding.